src/header/mod.rs

```diff
@@ -55,7 +56,7 @@ pub struct Header {
     /// Cryptographic algorithm to use
     pub alg: Option<Algorithm>,
     /// Critical headers to be understood
-    pub crit: Vec<RegisteredLabel<iana::HeaderParameter>>,
+    pub crit: Vec<RegisteredLabelWithPrivate<iana::HeaderParameter>>,
```

Comment:
This is a breaking change so please could you add a new `## 0.4.0 - TBD` section in `CHANGELOG.md` with an entry describing the change.

Reply:
done